src/components/Button.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Button(props) {
+  const { name } = props;
+  return (
+    <>
+      Button name:
+      {name}
+    </>
+  );
+}
+
+Button.propTypes = {
+  name: PropTypes.string,
+};
+
+Button.defaultProps = {
+  name: '0',
+};
```

Comment:
- I don't think this validation is needed. You may remove it to make your codebase more clean.

Reply:
Yes, I agree. However, ESLint was detecting an error if I remove setting default value.
So, I had to **Validate** and give a `defaultProps` value.
Please see below snapshot for more details:
<img width="913" alt="Screen Shot 2021-07-12 at 15 38 38" src="https://user-images.githubusercontent.com/60043698/125274660-9ea86d80-e327-11eb-83a1-56f8e4b4d4d3.png">

Also, when I removed **Validation** and `defaultProps`, ESLint is also detecting additional error:
<img width="919" alt="Screen Shot 2021-07-12 at 15 45 33" src="https://user-images.githubusercontent.com/60043698/125275241-37d78400-e328-11eb-86e2-1a280d32cf9c.png">